src/Task1.js

```diff
@@ -0,0 +1,27 @@
+import './Task1.css'
+import { useState } from 'react';
+
+const Task1= () =>
+{
+    const [input,setinput]= useState(null);
+    const [name,setname]= useState(input)
```

Comment:
You can use only 1 state for this as the second state defined i.e. name itself is getting the value of input.